.gitignore

```diff
@@ -7,3 +7,6 @@ dist/
 
 # mypy
 .mypy_cache/
+
+# uv
+uv.lock
```

Comment:
Why this the project didn't use uv!

Reply:
I'll remove this.

Reply:
Removed in 7277daa0a83c7c6f7e8d9a11205649823fa90cc6.